From d217a0d085ee896243691ce90e3fb0932e5de6f5 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Wed, 8 Oct 2025 10:54:32 +0200
Subject: [PATCH] arm64: defconfig: enable PCIe EP doorbell support

Add PCI_ENDPOINT_MSI_DOORBELL in fragment-02-defconfig-addons.config

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I407de974eec8288d41c186201450d7df9f81e553
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/497595
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -62,6 +62,7 @@ CONFIG_OF_OVERLAY=y
 CONFIG_PCIEASPM_POWERSAVE=y
 CONFIG_PCIE_STM32=m
 CONFIG_PCIE_STM32_EP=m
+CONFIG_PCI_ENDPOINT_MSI_DOORBELL=y
 CONFIG_PHY_STM32_COMBOPHY=m
 CONFIG_PHY_STM32_USB2PHY=m
 CONFIG_PINCTRL_STM32_HDP=m
